test/CMakeLists.txt

```
@@ -17,11 +17,12 @@ add_library(valhalla_test
   )
 target_include_directories(valhalla_test PUBLIC
   ${VALHALLA_SOURCE_DIR}/test
-  ${VALHALLA_SOURCE_DIR}/test/gurka
-  ${VALHALLA_SOURCE_DIR}/third_party/protozero/include
+  ${VALHALLA_SOURCE_DIR}/test/gurka)
+target_include_directories(valhalla_test PUBLIC SYSTEM
```

```suggestion
target_include_directories(valhalla_test SYSTEM PUBLIC
```

I think these are misordered: https://cmake.org/cmake/help/latest/command/target_include_directories.html